### ISSUE TITLE
Periodic deletion of Bulkrax /imports directory

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-curate#2091]
In order to prevent Curate from running out of disk space ([1/18/23 occurrence](https://eulcoreservices.slack.com/archives/GA9R6ADNH/p1674061736724589)), we need to expand our deletion/cleanup jobs to periodically to clean the /imports directory where we upload files to be used for Bulkrax ingests.

To investigate:

1. What is stored in the /working directory and if it should be periodically cleared
2. How to clear /imports which are stored per release

We currently have an AWX job that clears out the /tmp directory:

- Curate-Prod EBS Local Volume Subdirectory Clean

Recommended frequency: same as the above job

**Comment by bwatson78** [comment]
Questions: 

- Is `/tmp` on the local or the mount?
- For @eporter23, moving `/imports` would mean broken paths to old imports, is that ok?

**Comment by eporter23** [comment]
Yes, that is fine, but will that impact @jcrompton42 's file copy process to move files from EFS to /imports for Bulkrax?

**Comment by bwatson78** [comment]
It would just need to be updated.

**Comment by jcrompton42** [comment]
What @bwatson78 said, when the /imports location changes, can someone make me a ticket to update that?

**Comment by bwatson78** [comment]
@jcrompton42 Yup, I will.

**Comment by bwatson78** [comment]
Notes:
- Removing the ENV of `'WORKING_PATH'` would change `/working` to the default of `/tmp/uploads`, which currently doesn't exist.
- If I unset `config.import_path = 'imports'` in bulkrax' initializer, it will default out the import folder to `/tmp/imports`.

**Comment by bwatson78** [comment]
@rotated8 Knowing all of the info above, should I move forward with bringing both to `/tmp` in Arch as a trial run?

**Comment by eporter23** [comment]
@bwatson78 tested work in Curate v2.7.0 by running an export from test and a manually uploaded zip file import into arch. Import worked fine.

**Comment by eporter23** [comment]
@bwatson78 related to the above comment, when I [exported from test](https://curate-test.library.emory.edu/exporters/27?locale=en) I noticed the CSV had a couple of those extraneous multi-valued field columns again for abstract and publisher. ![Screen Shot 2023-03-02 at 1.08.39 PM.png](https://images.zenhubusercontent.com/5b6de3fda32a4f476097b0be/c4133695-614e-4dd1-a453-5523ccd45391) 
![Screen Shot 2023-03-02 at 1.08.49 PM.png](https://images.zenhubusercontent.com/5b6de3fda32a4f476097b0be/fdb250cc-fdb4-439a-9894-7b00600de2f7)

**Comment by eporter23** [comment]
@bwatson78 @jcrompton42 do we need another ticket related to updating the EFS to Curate file copy job?

**Comment by jcrompton42** [comment]
I updated the file copy job so we should be good on that.

**Comment by eporter23** [comment]
Notes from Slack discussion 10/4/23: this is implemented in prod, but each release has its own tmp/imports folder so as we do new releases we leave some stuff hanging out in imports.

**Comment by eporter23** [comment]
I'm going to go ahead and close this one, but will refer back to it if we run into again (we do seem to occasionally run out of disk space, but that maybe a separate issue).
